parity-util-mem/src/malloc_size.rs

```diff
@@ -68,6 +68,7 @@ use std::sync::Arc;
 pub use alloc::boxed::Box;
 #[cfg(not(feature = "std"))]
 use core::ffi::c_void;
+#[cfg(feature = "std")]
```

Comment:
is it required though?

Reply:
Yes, there are bounds in implementation for HashSet